.pylintrc

```diff
@@ -138,63 +130,33 @@ property-classes=abc.abstractproperty
 # Regular expression matching correct module names
 module-rgx=(([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+))$
 
-# Naming hint for module names
-module-name-hint=(([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+))$
-
 # Regular expression matching correct constant names
 const-rgx=(([A-Z_][A-Z0-9_]*)|(__.*__))$
 
-# Naming hint for constant names
-const-name-hint=(([A-Z_][A-Z0-9_]*)|(__.*__))$
-
 # Regular expression matching correct class names
 class-rgx=[A-Z_][a-zA-Z0-9]+$
```

Comment:
Since pylint enforces PEP8 name conventions by default, do we need these custom regexes? Are there names we're using now that don't comply with PEP8?

Reply:
I think this is a case of generating the default pylintrc with pylint and then modifying it, so these custom regexes just match the pylint default. We can try removing more stuff from the file. I just removed what I did because pylint was complaining that those options were no longer valid.

pylint has changed some defaults and removed some checkers since version 2.7 to be more in line with things like black.